third_party/blink/renderer/modules/cobalt/window_h_5_vcc.idl

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Extends the window interface.
+[
+  ImplementedAs=H5vcc,
+  SecureContext
+] partial interface Window {
+  [SameObject] readonly attribute H5vcc h5vcc;
```

Comment:
Should this be `H5Vcc`? Since the camel case file name is "...h_5_vcc".  IOW to produce `H5vcc`, shouldn't the file name be `...h_5vcc.idl`?

Reply:
Same as above, I think that's reasonable (though more ideal would probably just be renaming it), but right now we're just mimicking c25 names.